OpenRA.Mods.Common/Traits/Player/PlaceBuilding.cs

```diff
@@ -23,10 +23,10 @@ public class PlaceBuildingInit : IActorInit { }
 	public class PlaceBuildingInfo : ITraitInfo
 	{
 		[Desc("Palette to use for rendering the placement sprite.")]
-		[PaletteReference] public readonly string Palette = TileSet.TerrainPaletteInternalName;
+		[PaletteReference] public readonly string Palette = "terrain";
```

Comment:
How are these going to work if you have a tileset that doesn't use the "terrain" palette?

Reply:
Uh.. the same way as before? the default value for those traits is not changed, and can be overridden like before.